src/core/toga/widgets/optioncontainer.py

```diff
@@ -56,10 +153,27 @@ def add(self, label, widget):
         widget.app = self.app
         widget.window = self.window
 
-        self._content.append(widget)
+        self._content.append(label, widget)
         self._impl.add_content(label, widget._impl)
         widget.refresh()
 
+    def remove(self, index):
+        disabled_siblings = not any(
+            [
+                opt.enabled for opt in self._content
+                if opt != self._content[index]
+            ]
+        )
+
+        if len(self._content) == 1 or disabled_siblings:
+            # if sibling options are disabled or there is
+            # only one tab in the option container, don't allow
+            # remove tab.
+            return
```

Comment:
In this case "don't allow" should probably be an error; If I try to disable something, and it can't be disabled, I wouldn't expect that API to fail silently.